oncogene.js

```diff
@@ -257,8 +261,8 @@ class Oncogene {
                     throw new Error(`Variant ${variantNum} of step ${stepNum} should have hint or code`)
                 }
 
-                if (this.isUndefined(variant.value)) {
-                    throw new Error(`Variant ${variantNum} of step ${stepNum} should have a value`)
+                if (this.isUndefined(variant.value) && (this.isUndefined(variant.skip) || variant.skip)) {
```

Comment:
`variant.skip == true` shouldn't cause error.

Try this: `if (this.isUndefined(variant.value) && !variant.skip)`